.github/workflows/build.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
+      - uses: actions/checkout@v4
```

Comment:
Based on its previous PRs, won't Renovate revert this unless we use the full dot version like we do for `setup-java`?

Reply:
It will keep whatever format you're using. I'm not sure why it glitched and switched a bunch of repos to SHAs, but I switched them all back and none have seen follow-up PRs from it.